packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -66,17 +85,48 @@ const DestroyButton: React.FC<{ name: string }> = ({ name }) => {
   `)
 
   return (
-    <button
-      aria-label={`Delete ${name}`}
-      onClick={(): void => {
+    <Button
+      variant="SECONDARY"
+      size="S"
+      sx={{
+        paddingX: 5,
+        paddingY: 4,
+        color: `white`,
+      }}
+      onClick={(evt): void => {
+        evt.preventDefault()
         deleteGatsbyPlugin({ name })
       }}
     >
-      X
-    </button>
+      Uninstall
+    </Button>
   )
 }
 
+const SectionHeading: React.FC<HeadingProps> = props => (
+  <Heading sx={{ color: `white`, fontWeight: `500` }} {...props} />
+)
+
+const PluginCard: React.FC<{ name: string }> = ({ name }) => (
+  <Flex
+    flexDirection="column"
+    gap={7}
+    sx={{ backgroundColor: `grey.80`, padding: 5, borderRadius: 2 }}
+  >
+    <Text size="L" sx={{ color: `white`, fontWeight: `500` }}>
+      {name}
+    </Text>
+    <Text sx={{ color: `grey.40` }}>
+      Start setting up your sites styles with one of our curated recipes to
+      write styling the way you love. Choose from libraries like theme-UI,
+      emotion, and styled-components.
```

Comment:
I think this is okay for MVP, but we should audit the UI microcopy with design and learning team

Reply:
This isn't copy, this is a placeholder for the description of a plugin 😬